src/handwritten/task.py

```diff
@@ -402,6 +402,39 @@ def is_task_done(self):
 
         return is_task_done
 
+    def perform_bridge_offset_nulling_cal(self, channel=""):
+        """
+        Performs a bridge offset nulling calibration on the channels in the task.
+        If the task measures both bridge-based sensors and non-bridge-based sensors,
+        use the channels input to specify the names of the channels that measure 
+        bridge-based sensors.
+        """
+
+        self._interpreter.perform_bridge_offset_nulling_cal(
+            self._handle, channel)
+
+    def perform_bridge_offset_nulling_cal_ex(self, channel="", skip_unsupported_channels=False):
+        """
+        Performs a bridge offset nulling calibration on the channels in the task.
+        If the task measures both bridge-based sensors and non-bridge-based sensors,
+        use the channels input to specify the names of the channels that measure 
+        bridge-based sensors.
+        """
+
+        self._interpreter.perform_bridge_offset_nulling_cal_ex(
+            self._handle, channel, skip_unsupported_channels)
+
+    def perform_thrmcpl_lead_offset_nulling_cal(self, channel="", skip_unsupported_channels=False):
+        """
+        Performs thermocouple lead offset nulling calibration on the channels in 
+        the task to compensate for offsets introduced by open thermocouple detection.
+        Keep the measured temperature as constant as possible while performing this 
+        adjustment.
+        """
+
+        self._interpreter.perform_thrmcpl_lead_offset_nulling_cal(
+            self._handle, channel, skip_unsupported_channels)
```

Comment:
This might need to be handwritten because of my comment about not wanting `_ex`, but if it doesn't have to be, that would be nice. We can talk offline about whether or not that is the case.